console/frontend/src/main/frontend/src/app/views/ibisstore-summary/ibisstore-summary.component.html

```diff
@@ -79,20 +79,22 @@ <h4>Result</h4>
                 </tr>
               </thead>
               <tbody *ngFor="let t of result">
-                <tr *ngFor="let s of t.slots">
-                  <td
-                    *ngIf="s.first"
-                    [attr.rowspan]="t.slotcount"
-                    title="{{ t.name }}"
-                  >
-                    {{ t.type }}
-                  </td>
-                  <td *ngIf="s.first" [attr.rowspan]="t.slotcount">
-                    {{ t.msgcount }}
-                  </td>
+                <tr *ngFor="let s of t.slots; let first = first">
+                  <ng-container *ngIf="first; else notFirstSlot">
```

Comment:
Voor de volgende keer 😄 
```suggestion
                <tr *ngFor="let s of t.slots; let firstElement = first">
                  <ng-container *ngIf="firstElement; else notFirstSlot">
```